Userland/Libraries/LibWeb/HTML/Worker.idl

```diff
@@ -0,0 +1,21 @@
+[Exposed=(Window)]
+interface Worker : EventTarget {
+  constructor(USVString scriptURL, optional WorkerOptions options = {});
+
+  undefined terminate();
+  //undefined postMessage(any message, sequence<any> transfer);
+  //undefined postMessage(any message, optional StructuredSerializeOptions options = {});
+
+  attribute EventHandler onmessage;
+  attribute EventHandler onmessageerror;
+};
+
+dictionary WorkerOptions {
+  USVString type = "classic";
+  USVString credentials = "same-origin";
+  DOMString name = "";
+};
+
+dictionary StructuredSerializeOptions {
+  sequence<any> transfer = [];
+};
```

Comment:
Perhaps we could add a comment that `StructuredSerializeOptions` originally comes from the MessagePort idl?

Reply:
This is `sequence<object> transfer = [];` in the spec?

Reply:
Also, isn't it unused since this only has the first postMessage overload?

Reply:
That is correct. Removed it for now.